### PR TITLE
mining and smiting update

### PR DESCRIPTION
**Pull request by Connor3455**

### CODE REVIEW

File: rs3-chunkpicker-chunkinfo-export.json [hunk 1]
```diff
@@ -47310,6 +47339,13 @@
 				"NoPet": true,
 				"NoBoost": true
 			},
+			"Imbue ~|chaos-infused moonstone)|~": {
```
Comment (ThaFrenchFry) [inline rs3-chunkpicker-chunkinfo-export.json]: parenthesis?

File: rs3-chunkpicker-chunkinfo-export.json [hunk 2]
```diff
@@ -8429,6 +8429,14 @@
 				"NoPet": true,
 				"NoBoost": true
 			},
+			"Wield the ~|Masterwork 2h sword|~": {
```
Comment (SnowyToo) [inline rs3-chunkpicker-chunkinfo-export.json:8432]: Should not be capitalized

File: rs3-chunkpicker-chunkinfo-export.json [hunk 3]
```diff
@@ -22000,6 +22008,13 @@
 				"Priority": 7,
 				"Not F2P": true
 			},
+			"Craft a ~|Faceted chaos-infused moonstone|~": {
```
Comment (SnowyToo) [inline rs3-chunkpicker-chunkinfo-export.json:22011]: Should not be capitalized

File: rs3-chunkpicker-chunkinfo-export.json [hunk 4]
```diff
@@ -23238,7 +23253,7 @@
 				"Output": "Leviathan ring",
 				"Primary": false,
 				"Priority": 13
-			},
+			},
```
Comment (SnowyToo) [inline rs3-chunkpicker-chunkinfo-export.json]: Remove indentation after },

File: rs3-chunkpicker-chunkinfo-export.json (hunk 1 above)
Comment (SnowyToo) [inline rs3-chunkpicker-chunkinfo-export.json]: Bracket

File: rs3-chunkpicker-chunkinfo-export.json [hunk 5]
```diff
@@ -47909,6 +47945,13 @@
 				"Objects": ["Ore[+]"],
 				"Primary": false,
 				"Priority": 5
+			},
+				"Mine with an ~|primal pickaxe|~": {
```
Comment (SnowyToo) [inline rs3-chunkpicker-chunkinfo-export.json]: Indentation

File: rs3-chunkpicker-chunkinfo-export.json [hunk 6]
```diff
@@ -48323,6 +48366,76 @@
...
+				"Objects": ["Marmaros rock"],
+				"Primary": true,
+				"Output": "Marmaros ore",
+				"Priority": 3
+			},
+			"Mine a ~|kratonium rock|~": {
+				"Level": 100,
+				"Objects": ["Kratonium rock"],
+				"Primary": true,
+				"Output": "kratonium ore",
```
Comment (ThaFrenchFry) [inline rs3-chunkpicker-chunkinfo-export.json:48394]: capitalize

File: rs3-chunkpicker-chunkinfo-export.json [hunk 7]
```diff
@@ -47909,6 +47945,13 @@
 				"Objects": ["Ore[+]"],
 				"Primary": false,
 				"Priority": 5
+			},
+				"Mine with an ~|primal pickaxe|~": {
+				"Items": ["Elderpick[+]"],
```
Comment (SnowyToo) [inline rs3-chunkpicker-chunkinfo-export.json]: Should be Primalpick[+] (not added)

File: rs3-chunkpicker-chunkinfo-export.json [hunk 8]
```diff
@@ -48323,6 +48366,76 @@
...
+				"Objects": ["Argonite rock"],
+				"Primary": true,
+				"Output": "Argonite ore",
+				"Priority": 7
+			},
+			"Mine a ~|katagon rock|~": {
+				"Level": 100,
+				"Objects": ["Katagon rock"],
+				"Primary": true,
+				"Output": "Katagon ore",
```
Comment (ThaFrenchFry) [inline rs3-chunkpicker-chunkinfo-export.json:48422]: capitalize

File: rs3-chunkpicker-chunkinfo-export.json [hunk 9]
```diff
@@ -59400,6 +59560,94 @@
...
+			},
+			"Forge a ~|unsharpened masterwork blade|~": {
+				"Items": ["Shaped masterwork blade*", "Masterwork sword trim*"],
+				"Level": 110,
+				"Objects": ["Forge", "Anvil[+]"],
+				"Output": "Unsharpened masterwork blade",
+				"Primary": true,
+				"Priority": 11
+			},
+				"Forge a ~|sharpened masterwork blade|~": {
```
Comment (SnowyToo) [inline rs3-chunkpicker-chunkinfo-export.json:59635]: Indentation

File: rs3-chunkpicker-chunkinfo-export.json [hunk 10]
```diff
@@ -114165,6 +114437,18 @@
 				"Items": ["Red dye", "Blue dye"],
 				"Output": "Purple dye"
 			},
+			"Make a brown dye*": {
+				"Items": ["Yellow orange", "Purple dye"],
+				"Output": "Brown dye"
+			},
+			"Make a brown dye*": {
```
Comment (SnowyToo) [inline rs3-chunkpicker-chunkinfo-export.json]: Don't have multiple tasks with the exact same title

File: rs3-chunkpicker-chunkinfo-export.json [hunk 11]
```diff
@@ -114165,6 +114437,18 @@
...
+				"Items": ["Red orange", "Blue dye"],
+				"Output": "Brown dye"
+			},
+			"Make a brown dye*": {
```
Comment (SnowyToo) [inline rs3-chunkpicker-chunkinfo-export.json:114448]: Don't have multiple tasks with the exact same title

File: rs3-chunkpicker-chunkinfo-export.json [hunk 12]
```diff
@@ -139254,13 +139544,17 @@
...
-				"Abandoned 1 barrier": 1,
-				"Furnished barrier": 1
-			},
+		    "Object": {
+		        "Free-for-all barrier": 2,
+		        "Broken-stairs": 1,
+		        "Frozen barrier": 1,
+		        "Abandoned 1 barrier": 1,
+		        "Promethium rock": 5,
+				"Fractite rock": 5,
```
Comment (SnowyToo) [inline rs3-chunkpicker-chunkinfo-export.json:139553]: Indentation (x3)

Comment (ThaFrenchFry) [inline rs3-chunkpicker-chunkinfo-export.json:139553]: indents

**Review by SnowyToo — CHANGES_REQUESTED** [review]

**Review by ThaFrenchFry — CHANGES_REQUESTED** [review]

**Review by ThaFrenchFry — APPROVED** [review]

**Review by SnowyToo — APPROVED** [review]

**Pull request merged**